src/plugins/console/public/application/components/split_panel/resizer.tsx

```diff
@@ -24,7 +24,7 @@ export function Resizer(props: Props) {
     <button
```

Comment:
Please, move the `Resizer` component to `containers/split_panel`, as it is used only there. This style is allowed by the other components, placed at the `containers` folder.